docs/how-to-enable-new-languages.md

```diff
@@ -281,3 +281,101 @@ Once these are in place, you should be able to run `npm run develop` to view you
+
+The contents will look something like this:
+
+```yaml
+article0title: 'Learn JavaScript'
+article0link: 'https://www.freecodecamp.org/news/learn-javascript-free-js-courses-for-beginners/'
+article1title: 'Linux ln Example'
+article1link: 'https://www.freecodecamp.org/news/linux-ln-how-to-create-a-symbolic-link-in-linux-example-bash-command'
+article2title: 'JS document.ready()'
+article2link: 'https://www.freecodecamp.org/news/javascript-document-ready-jquery-example/'
+article3title: ...
+article3link: ...
+  ...
+```
+
+Open a PR to the CDN repo with these changes and tag `@freeCodeCamp/dev-team` for review.
+
+## Prep the News Repo for the New Language
+
+### Modify the Main Config File
+
+The [News repo](https://github.com/freeCodeCamp/news) pulls data from a Ghost instance, builds the site, and deploys it.
+
+Clone the News repo and create a new branch.
+
+Open the `config/index.js` file to add the new language and configure the necessary values. There are a few objects and arrays to modify:
+
+- `locales`: This array contains the active and upcoming News languages. These are the values that are used in the `.env` file to choose the Ghost instance and UI to use for each build. Add the text name of the new language in lowercase to this array.
+- `localeCodes`: This object is a map of ISO codes for each language, and is used to configure i18next before building the UI. To add a new language, use the lowercase language name as the _key_ and the ISO 639-1 language code as the _value_.
+- `algoliaIndices`: This object is a map of Algolia indicies for each language. To add a new language, use the lowercase language name as the _key_, and `news-` followed by the lowercase ISO 639-1 language code as the _value_.
+
+> [!NOTE]
+> If you are unsure about the string to use while setting `algoliaIndicies, send a message to Kris (@scissorsneedfoodtoo), or someone else with access to Algolia, and ask them to check.
+
+For example, if you are launching Dothraki News, here are what the objects / arrays above should look like:
+
+```js
+const locales = ['arabic', 'bengali', 'chinese', 'english', 'dothraki'];
+
+const localeCodes = {
+  arabic: 'ar',
+  bengali: 'bn',
+  chinese: 'zh',
+  english: 'en',
+  dothraki: 'do'
+};
+
+const algoliaIndices = {
+  arabic: 'news-ar',
+  bengali: 'news-bn',
+  chinese: 'news-zh',
+  english: 'news',
+  dothraki: 'news-do'
+};
+```
+
+> [!NOTE]
+> Note that the values above are in alphabetic order. Also, Dothraki does not actually have an ISO 639-1 code, so `do` is just used as an example.
+
+### Add the i18next JSON Files for the New Language
+
+Next, go to the `config/i18n/locales` directory, create a new folder, and give it the name of the new language you're adding. For example, if you're launching Dothraki News, create a new folder named `dothraki`.
+
+Then go to the `english` directory, copy the `links.json` and `meta-tags.json` files, and paste them into your new folder.
```

Comment:
```suggestion
Then go to the `english` directory, copy the JSON files, and paste them into your new folder.
```

As mentioned on the Indonesian news PR.

Reply:
Sure, this looks good, and should be easier to follow.

I'll just make sure to update the following section and say that the contents of the English `serve.json` file should be replaced by the object with the empty `redirects` array below.